js/my-github.js

```diff
@@ -1,2 +1,35 @@
 // Write code here to communicate with Github
+function getRepos(username){
+    document.getElementById('repos-list').innerHTML = '';
 
+    fetch("https://api.github.com/users/" + username + "/repos").then(
+        function(response) {
+            return response.json();
+        }
+    ).then(
+        function(repos) {
+            reposArray = [];
+            for (i = 0; i < repos.length; i++) {
+            reposArray = reposArray.concat([[repos[i]["name"], repos[i]['html_url']]]);
```

Comment:
I would use `map` here since you are creating a new array from the original, with the same number of elements, and each element only depends on the one in the original array. When you use a `for` loop, none of these things are obvious without looking in detail at the code - it's more fragile. In general I would use one of the array methods we've seen whenever you can.